src/css/nav.css

```diff
@@ -270,14 +194,20 @@ html.is-clipped--nav {
   margin: 0;
   padding: 0;
   width: 100%;
+  color: var(--color-aliases-static-palette-text-secondary);
```

Comment:
Could we add a dark-mode version of this? It's hard to read in dark mode. Same goes for the breadcrumbs

![2024-07-29_11-16-48](https://github.com/user-attachments/assets/caef17bb-cca9-4e3d-8671-d665e344fa42)

Reply:
I think we could do a quick-pass on the dark mode as a followup, there's a few bits and pieces that don't look the best.